python/Quickstart.MD

```diff
@@ -142,192 +119,201 @@ Body of event request:
+
+**Type or copy the following into your Python terminal**  
+```
+request_body_json = json.dumps(request_body)
 ```
+Note that we have omitted the `indent=4, sort_keys=False` options from the `json.dumps()` command that we included in the print statement.  We used these for clarity when print, but are not necessary when creating our JSON string.
 
 ## Send Request
+Once your request header and request body are complete you are ready to submit your request.  
+You will need the URL for the APT-SCWM environment.  Contact your administrator for this information.  
+APT-SCWM uses and event driven API that communicates using the POST verb of the request/response model.  Create a variable to hold the response so you can work with your results.  
 
-Once you have a request header, body header, and body payload you are ready to submit your request.
-
-The `utils.py` file contains a function to send your request and capture the response.
-
-An example Python terminal session:
+**Type or copy the following into your Python terminal**  
 ```
->>> import utils
->>> my_file = 'addIncident.json'
->>> my_token = token_received_from_other_processes
-
->>> event_data = {"eventName": "agile-process-teams:add-direct-supplier-incident:v2",
-                  "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
-                  "processNetworkId": "945feb8e-09a7-4fef-9a8a-b3c5b56f87d"}
->>> request_headers = utils.create_headers(my_token)
->>> payload_data = utils.read_payload_file(my_file)
->>> request_body = utils.create_payload(event_data, payload_data)
->>> result = utils.post_request(request_headers, request_body)
+import requests
+url = "URL_FROM_YOUR_ADMINISTRATOR"
+result = requests.request("POST", url, headers=request_header, data=request_body_json)
 ```
 An example successful response:
 ```
 {
     "header": {
         "headerVersion": 1,
-        "eventName": "agile-process-teams:add-direct-supplier-incident-response:v2",
-        "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
+        "eventName": "agile-process-teams:add-direct-supplier-incident-response:v3",
+        "ownerId": "YOUR_OWNER_ID",
         "isErr": false,
         "errCode": "200_OK",
-        "licensePlate": "qL0BJs"
+        "licensePlate": "LICENSE_PLATE_ID"
     },
     "payload": {
-        "id": "fd04dd70-c162-43ef-8118-539ca01de471"
+        "id": "ITEM_ID"
     }
 }
 ```
 
 ## Response Handling
 
-The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a string, not a JSON object. To get this identifier, enter the following in your Python terminal:
-```
->>> import json
->>> response_json = json.loads(result.text)
->>> incident_id = response_json.get('payload').get('id')
-```
+The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a JSON object that has been converted to a string for transmission. To work with this data, convert it to a Python dictionary.  
```

Comment:
The `id` in a successful response payload is the identifier of the created incident. Notice that the response is a JSON object that is converted to a string for transmission. To work with this data, convert it to a Python dictionary.

Reply:
Fixed